frontend/src/pages/shipments/ShipmentList.tsx

```diff
@@ -61,13 +61,13 @@ const ShipmentList: FunctionComponent = () => {
 
   return (
     <LayoutWithNav>
-      <div className="max-w-5xl mx-auto border-l border-r border-gray-200 min-h-content">
-        <header className="p-6 border-b border-gray-200 flex items-center justify-between">
-          <h1 className="text-navy-800 text-3xl">Shipments</h1>
-          <ButtonLink to={ROUTES.SHIPMENT_CREATE}>Create a shipment</ButtonLink>
+      <div className="max-w-5xl mx-auto bg-white border-l border-r border-gray-200 min-h-content">
+        <header className="p-6 border-b border-gray-200 md:flex items-center justify-between">
+          <h1 className="text-navy-800 text-3xl mb-4 md:mb-0">Shipments</h1>
+          <ButtonLink to={ROUTES.SHIPMENT_CREATE}>Create shipment</ButtonLink>
         </header>
-        <main>
-          <table className="w-full" {...getTableProps()}>
+        <main className="pb-20 overflow-x-auto">
+          <table className="w-full whitespace-nowrap" {...getTableProps()}>
```

Comment:
`overflow-x-auto` and `whitespace-nowrap` allow tables to overflow horizontally on mobile instead of getting squished and make the _whole page_ overflow.